Stonks/Forms/FormSmartSaver.cs

```diff
@@ -40,8 +40,9 @@ public void setTrackBarValues()
             //Setting the values to represent the track bars
             if (housingExpense != null) {
                 trackBarHousing.Maximum = Convert.ToInt32(max);
-                trackBarHousing.Value = Convert.ToInt32(housingExpense.Value);
-                labelHousingExpense.Text = (housingExpense.Value).ToString("€#.#");
+                trackBarHousing.Value = Convert.ToInt32(housingExpense.PlannedValue); //Fix if value becomes higher than expense(just like savings?)
+                labelHousingExpense.Text = (housingExpense.PlannedValue).ToString("€#.#");
+                labelHousingExpensesActual.Text = (housingExpense.Value).ToString("€#.#");
```

Comment:
Not sure if it's the right place to have both planned and actual expenses shown. Isn't the Progress Form to do that?